packages/sophos/data_stream/utm/elasticsearch/ingest_pipeline/default.yml

```
@@ -1,87 +1,168 @@
-      ignore_missing: true
-  # User agent
-  - user_agent:
-      field: user_agent.original
-      ignore_missing: true
-  # IP Geolocation Lookup
-  - geoip:
-      field: source.ip
-      target_field: source.geo
-      ignore_missing: true
-  - geoip:
-      field: destination.ip
-      target_field: destination.geo
-      ignore_missing: true
-
-  # IP Autonomous System (AS) Lookup
-  - geoip:
-      database_file: GeoLite2-ASN.mmdb
-      field: source.ip
-      target_field: source.as
-      properties:
-          - asn
-          - organization_name
-      ignore_missing: true
-  - geoip:
-      database_file: GeoLite2-ASN.mmdb
-      field: destination.ip
-      target_field: destination.as
-      properties:
-          - asn
-          - organization_name
-      ignore_missing: true
-  - rename:
-      field: source.as.asn
-      target_field: source.as.number
-      ignore_missing: true
-  - rename:
-      field: source.as.organization_name
-      target_field: source.as.organization.name
-      ignore_missing: true
+      tag: set_ecs_version
   - rename:
-      field: destination.as.asn
-      target_field: destination.as.number
+      field: message
+      target_field: event.original
       ignore_missing: true
+      tag: rename_event_original
+  - set:
+      field: event.category
+      value: [network]
+      tag: set_event_category
+  - set:
+      field: event.type
+      value: [info]
+      tag: set_event_type
+  - set:
+      field: event.kind
+      value: event
+      tag: set_event_kind
+
+  # Parse the event header: populates the timestamp, hostname, and process info
+  # from the header and stores the message in _tmp.raw_data
+  - grok:
+      field: event.original
+      if: ctx.event?.original != null
+      pattern_definitions:
+        SOPHOS_TIMESTAMP: (?:%{YEAR}:%{MONTHNUM}:%{MONTHDAY}-%{HOUR}:%{MINUTE}:%{SECOND})
+      patterns:
+        - "^%{SOPHOS_TIMESTAMP:_tmp.timestamp} (?:%{HOSTNAME:host.hostname}) %{DATA:process.name}(?:\\[%{POSINT:process.pid:long}\\])?: %{GREEDYDATA:_tmp.raw_data}"
+      tag: grok_syslog_header
+      on_failure:
+        - append:
+            field: error.message
+            value: "fail-{{{ _ingest.on_failure_processor_tag }}}"
+        - fail:
+            message: "Processor {{ _ingest.on_failure_processor_type }} with tag {{ _ingest.on_failure_processor_tag }} in pipeline {{ _ingest.on_failure_pipeline }} failed with message: {{ _ingest.on_failure_message }}"
+  # Process timestamp and timezone if configured (default UTC)
   - rename:
-      field: destination.as.organization_name
-      target_field: destination.as.organization.name
+      if: ctx._conf?.tz_offset != null
+      field: _conf.tz_offset
+      target_field: event.timezone
       ignore_missing: true
+      tag: rename_conf_timezone
+  - set:
+      field: event.timezone
+      value: UTC
+      override: false
+      tag: set_default_timezone
+  - date:
+      if: ctx?._tmp?.timestamp != null
```

```suggestion
      if: ctx._tmp?.timestamp != null
```
`s/\bctx\?\./ctx./g`